### ISSUE TITLE
TypeError on handrail.handrail() call

### ISSUE THREAD

**Issue by mmmveggies** [brekk/handrail#20]
Getting `a is not a function` using a few signatures for `handrail`

```javascript
const handrail = require('handrail')
handrail.handrail(1, () => {}, () => {}, () => {}) // throws
handrail.handrail(1, () => {}, () => {}, []) // throws also
handrail.handrail(() => {}, () => {}, () => {}, () => {}) // throws also
/*
TypeError: a is not a function
    at /project/node_modules/f-utility/f-utility.js:1:6947
    at filter$4 (/project/node_modules/f-utility/f-utility.js:1:4303)
    at filter$1 (/project/node_modules/f-utility/f-utility.js:1:4405)
    at /project/node_modules/f-utility/f-utility.js:1:4670
    at c (/project/node_modules/f-utility/node_modules/katsu-curry/katsu-curry.js:1:824)
    at a (/project/node_modules/f-utility/node_modules/katsu-curry/katsu-curry.js:1:679)
    at /project/node_modules/f-utility/f-utility.js:1:6921
    at c (/project/node_modules/f-utility/node_modules/katsu-curry/katsu-curry.js:1:824)
    at a (/project/node_modules/f-utility/node_modules/katsu-curry/katsu-curry.js:1:679)
    at /project/node_modules/f-utility/node_modules/katsu-curry/katsu-curry.js:1:1334
*/
```

Not exactly sure where this, other than in `f-utilities` somewhere. Will take a look

**Comment by brekk** [comment]
Hmm, will investigate.

Also @mmmveggies which version are you using? There've been lots of recent changes.

**Comment by mmmveggies** [comment]
`1.2.0`

**Comment by mmmveggies** [comment]
```
handrail@^1.1.8:
  version "1.2.0"
  dependencies:
    entrust "^0.0.3"
    f-utility "^3.0.3"
    fantasy-eithers "^0.0.2"
    katsu-curry "^0.0.8"
    ramda "^0.24.0"
```

**Comment by brekk** [comment]
Cutting a new version in the next 30 or so.

**Comment by mmmveggies** [comment]
Thanks for the fix!

**Comment by brekk** [comment]
Should be fixed in `1.3.0`, which is publishing right now.
